django_project/api/api_views/v2/authentication.py

```diff
@@ -12,10 +12,31 @@ def authenticate(self, request):
         if not api_key:
             raise exceptions.AuthenticationFailed('api-key is needed')
         key = UserApiKey.get_key_from_api_key(api_key)
+        if not key:
+            raise exceptions.AuthenticationFailed(
+                'api-key is invalid, '
+                'please recreate using enrollment form on profile page.'
+            )
+        if not key.is_active:
+            raise exceptions.AuthenticationFailed(
+                'api-key is not active, please wait for admin approval.'
```

Comment:
This API key is not active. Please wait for it to be approved by the admins https://healthsites.io.

Reply:
Use an explicit link to the enrollment page if possible.